src/Core/Users/class.user.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface
+
+Class User extends UserInterface{
+
+	public function __construct($email, $password){
+		$this->setEmail($email);
+		$this->setPassword($password);
+	}
+	
+	public function setEmail($email){
+		$this->email = $email;
+	}
+	
+	public function setPassword($password){
+		$this->password = $password;
+	}
+	
+	public function checkPassword(){
+		$sql = "SELECT firstName,lastName,createdAt FROM user WHERE $email='".$this->email."' AND password='".$this->password."' ";
```

Comment:
### Issues
- Wrong method signature. Refer the interface docs.
- Wide open to sql injection. 

We are not concerned about persisting anything in `User` entity. Please remove all sqls